internal/services/instance_types_service_test.go

```diff
@@ -0,0 +1,70 @@
+package services_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	clientStub "github.com/RHEnVision/provisioning-backend/internal/clients/stubs"
+	"github.com/RHEnVision/provisioning-backend/internal/dao/stubs"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+	"github.com/RHEnVision/provisioning-backend/internal/services"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListInstanceTypes(t *testing.T) {
+
+	t.Run("with region", func(t *testing.T) {
+		var names []string
+		ctx := stubs.WithAccountDaoOne(context.Background())
+		ctx = identity.WithTenant(t, ctx)
+		ctx = clientStub.WithSourcesClient(ctx)
+		ctx = clientStub.WithEC2Client(ctx)
```

Comment:
I am really proud of us, look at what we've built :-)

I suggest, at some point not here, we should create some wrapper and simply:

```go
w := ctxstub.NewWrapper().WithTenant(ctx).WithSourcesClient().WithEc2Client()
ctxval.WithStub(t, ctx, w)
```

But this is good enough.